lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -9,67 +9,18 @@ module GitHelper
         # This typically commits and pushes:
         #  - The files in `./config/*` – especially `Version.*.xcconfig` files
         #  - The `fastlane/Deliverfile` file (which contains the `app_version` line)
-        #  - The `<ProjectRoot>/<ProjectName>/Resources/AppStoreStrings.pot` file, containing a key for that version's release notes
         #
         # @env PROJECT_ROOT_FOLDER The path to the git root of the project
-        # @env PROJECT_NAME The name of the directory containing the project code (especially containing the Resources/ subfolder)
         #
         # @param [Bool] include_deliverfile If true (the default), includes the `fastlane/Deliverfile` in files being commited
-        # @param [Bool] include_metadata If true (the default), includes the `fastlane/download_metadata.swift` file and the `.pot` file (which typically contains an entry or release notes for the new version)
         #
-        def self.commit_version_bump(include_deliverfile: true, include_metadata: true)
-          files_list = [File.join(ENV['PROJECT_ROOT_FOLDER'], 'config', '.')]
+        def self.commit_version_bump(include_deliverfile: true)
+          files_list = [File.join(get_from_env!(key: 'PROJECT_ROOT_FOLDER'), 'config', '.')]
           files_list.append File.join('fastlane', 'Deliverfile') if include_deliverfile
-          if include_metadata
-            files_list.append File.join('fastlane', 'download_metadata.swift')
-            files_list.append File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], 'Resources', ENV['APP_STORE_STRINGS_FILE_NAME'])
```

Comment:
> Seeing this, I wonder if it would be worth it to document somewhere the changes potentially needed in client projects when migrating to the next major version that will contain this change.

This is a nice idea! It's a good point on cases where there isn't necessarily a breaking change but still the client projects could do a clean up on unnecessary things they had to do, or a different behaviour they'd previously expect.
I think the examples you gave for this PR sound like a good starting point.

What about a `MIGRATION.md` file in the root, next to the `CHANGELOG.md`?
The way I see it, the structure and the process would be very similar to the change log, with sections for each version containing each a bullet list of items that require attention during the migration from the previous version. Then all PRs making such changes would include something in the list.
What do you think? I can kick it off for this PR, it shouldn't take long :-)